### DISCUSSION TITLE
It is posible to send query params when executing router.back()?

### DISCUSSION THREAD

**Discussion by jmacbhc** [vercel/next.js#53464] [App Router] [answered]
It's posible to send query params when using router.back()?

```
import {useRouter} = "next/navigation"

const router = useRouter()

router.back()
```

**Comment by Uy575** [comment, marked as answer]
Answer is no, In router.back() you can not send query params because it always navigate previous route which is in browser's history stack,
secondly they removed query now with useSearchParams() as replacement.

**Reply by jmacbhc** [reply]
Thaks for your answer.
